src/restic/node.go

```diff
@@ -496,9 +568,35 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 		err = errors.Errorf("invalid node type %q", node.Type)
 	}
 
+	switch runtime.GOOS {
+	case "windows", "openbsd":
+		return err
+	}
+	if err = node.fillAcls(path); err != nil {
+		return err
+	}
+
 	return err
 }
 
+func (node *Node) fillAcls(path string) error {
```

Comment:
Same as above, the function name `fillExtendedAttributes()` is probably better suited.